vcap/setup.py

```diff
@@ -1,18 +1,23 @@
 #!/usr/bin/env python3
 import os
+from pathlib import Path
 
 from setuptools import setup, find_namespace_packages
 
+# Get package version/metadata
+about = {}
+exec(Path("vcap/version.py").read_text(), about)
+
 test_packages = ["pytest", "mock"]
 
 PRE_RELEASE_SUFFIX = os.environ.get("PRE_RELEASE_SUFFIX", "")
 
 setup(
     name='vcap',
     description="A library for creating OpenVisionCapsules in Python",
-    author="Dilili Labs",
+    author="Aotu",
```

Comment:
More like Aotu.ai!! (Right? I don't know.)

Reply:
According to the branding page it's the "short name". But it seems to lean heavier on Aotu.ai branding, so I will change it to that, since this is more official than just talking about Aotu in passing.